src/telas/Post/index.js

```diff
@@ -30,69 +21,19 @@ export default function Post({ navigation, route }) {
     }
 
     async function atualizar() {
-        const data = { titulo, fonte, descricao, imagem: imagem ? imagem : null };
+        const data = { titulo, fonte, descricao };
         const resultado = await atualizarPost(item.id, data);
         if (resultado === "ok") {
-            verificarESalvarImagem(item.id);
             navigation.goBack();
         }
         else {
             Alert.alert("Erro ao atualizar");
```

Comment:
O problema é identificado em algum lugar? Para o usuário pode ser apenas um erro, mas o código também precisa identificar. Se for um erro conhecido, as vezes compensa indicar para o usuário o motivo, por exemplo, falta de conexão, falta de acesso etc